### PR TITLE
[Doc] 3.2-light-schema-change

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/zh/administration/Configuration.md [hunk 1]
```diff
@@ -509,6 +509,15 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 - 单位：秒
 - 默认值：86400
 
+##### fast_schema_evolution
+
+- 含义：是否开启集群内所有表的 fast schema evolution，取值：`TRUE`（默认） 或 `FALSE`。开启后增删列时可以提高 schema change 速度并降低资源使用。
+  > **NOTE**
```
Comment (evelynzhaojie) [inline docs/zh/administration/Configuration.md:515]: 少个空行，

**Review by amber-create — DISMISSED** [review]

File: docs/zh/administration/Configuration.md [hunk 2]
```diff
@@ -509,6 +509,15 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
...
+  > - StarRocks 存算分离集群不支持该参数。
+  > - 如果您需要为某张表设置该配置，例如关闭该表的 fast schema evolution，则可以在建表时设置表属性 [`fast_schema_evolution`](../sql-reference/sql-statements/data-definition/CREATE_TABLE.md#设置-fast-schema-evolution)。
```
Comment (evelynzhaojie) [inline docs/zh/administration/Configuration.md:517]: 可以 alter table 改吗

File: docs/en/administration/Configuration.md [hunk 3]
```diff
@@ -568,6 +568,15 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 - **Default**: 86400
 - **Description**: The timeout duration for the schema change operation (ALTER TABLE). Unit: seconds.
 
+#### fast_schema_evolution
+
+- **Default**: TRUE
+- **Description**: Whether to enable the fast schema evolution for all tables within the StarRocks cluster. Valid values are `TRUE` (default) and `FALSE`. Enabling this feature can increase the speed of schema changes and reduce resource usage when columns are added or dropped.
```
Comment (evelynzhaojie) [inline docs/en/administration/Configuration.md:574]: the fast schema evolution feature

File: docs/en/administration/Configuration.md [hunk 4]
```diff
@@ -568,6 +568,15 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
...
+  > **NOTE**
+  > - StarRocks shared-data clusters do not support this parameter.
+  > - If you need to configure the fast schema evolution for a specific table, such as disabling the fast schema evolution for a specific table, you can set the table property [`fast_schema_evolution`](../sql-reference/sql-statements/data-definition/CREATE_TABLE.md#set-fast-schema-evolution) at table creation.
```
Comment (evelynzhaojie) [inline docs/en/administration/Configuration.md:577]: the xxx feature

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Pull request merged**